### ISSUE TITLE
Uninstalling Classic Explorer doesn't revert Explorer defaults

### ISSUE THREAD

**Issue by MartinDerTolle** [Open-Shell/Open-Shell-Menu#981]
**Description:**
When I accidentaly installed Classic Explorer along with Open-Shell, I wanted to uninstall it, however when I did that, using the Open-Shell installer, my Explorer window looked like this:

![bugged explorer](https://user-images.githubusercontent.com/45734424/157739929-2815dd71-5640-4869-857f-697f4a1a5947.PNG)

Notice that the status bar and toolbar are now white. Changing the Windows theme or switching dark theme on-and-off doesn't fix it.

**Expected Behaviour:**
The Explorer window is recovered to the state it had before, which was a dark themed toolbar and status bar.

**Versions:**
 - Open-Shell 4.4.160
 - Windows 10 20H2

**Other:**
I suggest the Classic Explorer to be unticked in the installer by default, even if this gets fixed. Since it might not be expected to have it installed when Open-Shell is mainly known as an alternative to the default start menu.

**Comment by bonzibudd** [comment]
Are you using OldNewExplorer or another Explorer-editing program?

**Comment by MartinDerTolle** [comment]
No, but I have used third party themes with SecureUxThemepatcher before, but I was using the standard Windows theme (and no OldNewExplorer or such) when I had the problem. Also I'm using RetroBar, if that affects anything.

I already did a system check with SFC /scannow, but it didn't help.
If you know how I can restore Explorer back to normal, that'd be of much help!
I'm pretty sure it's something in the registry, because otherwise it should've been detected

**Comment by eliardneru** [comment]
same problem here, i also used uxthemepatcher, i uninstalled, disabled, but those buttons are still here
![image](https://user-images.githubusercontent.com/74941858/204745604-a9730590-8bda-49b5-a75c-6528df867293.png)

**Comment by ge0rdi** [comment]
@MartinDerTolle @eliardneru 
Does the issue persist also after machine reboot (or File Explorer process restart)?
`Open-Shell` doesn't do any persistent changes, so if it was properly uninstalled everything should return to original state.

Can you also check if `Open-Shell` was uninstalled properly?
You can check if there is `C:\Program Files\Open-Shell` folder and whether it contains some files (DLLs).
